src/pages/NotFound.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import {
-    View,
-    Text,
-    Image,
-    TouchableOpacity,
-    SafeAreaView,
+	View,
```

Comment:
You need to use 4 spaces for tabs e.g. if you use vs code see [here](https://code.visualstudio.com/docs/editor/codebasics#_indentation)

Reply:
Please fix in all files before requesting another review or the PR will be closed.

Reply:
Hi @marcaaron 

My Setting in VS Code :: 

{
	"editor.formatOnPaste": true,
	"editor.formatOnSave": true,
	"editor.fontSize": 18,
	"editor.tabSize": 4,
	"editor.insertSpaces": true,
	"editor.detectIndentation": false,
	"javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false
}

Please where is wrong ??

Reply:
https://code.visualstudio.com/docs/editor/codebasics#_indentation